policies/TrademarkPolicy.md

```diff
@@ -0,0 +1,217 @@
+    community, if the same is not true.
+2.  You cannot use Our Trademarks in a company or organization name
+    or as the name of a product or service.
+3.  You cannot use a name or logo that is confusingly similar to Our 
+    Trademarks.
+4.  You cannot use Our Trademarks in a domain name, with or without
+    commercial intent.
+5.  You cannot make merchandise with Our Trademarks for *commercial usage*.
+    In the case of commercial usage, where permission to use Our Trademarks 
+    has been provided by OpenSSL, we recommend that you truthfully advertise to 
+    customers which part of the selling price, if any, will be donated to the OpenSSL project.
+6.	In general you cannot use Our Trademarks in your own software branding 
+    or to describe or advertise your services or products relating to OpenSSL.  
+    However OpenSSL may grant permission to use Our Trademarks to indicate that
+    an OpenSSL product has been incorporated into another organisation's 
+    product or service. In these instances the use of conventions such as 
+	"compatible with" or "based On" will apply.
+ 
+
+## When Our Trademarks can be used without permission
+You may use Our Trademarks without prior written permission only in the 
+following circumstances:
+
+### Noncommercial and community usage
+
+Including but not limited to:
+* placing Our Trademarks on a personal web site or blog to support open 
+source software
+* making a page on a social networking web service to support open source 
+software
+* linking to the OpenSSL website to provide information or show support for 
+open source software
+
+As long as you are in compliance with the following conditions, no advance 
+written permission is necessary:
+
+1.  You use Our Trademarks to make true factual statements about
+    OpenSSL or communicate compatibility with your product truthfully using 
+    conventions such as "compatible with" or "based on".
+2.  Your intended use qualifies as "nominative fair use" of the OpenSSL
+    trademarks, i.e., merely identifying that you are talking about
+    OpenSSL in a text, without suggesting affiliation, sponsorship or 
+    endorsement.
+3.  You can use Our Trademarks to describe OpenSSL in articles,
+    titles or blog posts.
+4.  You can make t-shirts, desktop wallpapers, caps, or other
+    merchandise with Our Trademarks for *non-commercial usage*.
+
+
+### Commercial and Business usage
+As long as you are in compliance with the following conditions, no advance 
+written permission is necessary:
+1.  You use Our Trademarks to make true factual statements about
+    OpenSSL.
+2.  Your intended use qualifies as "nominative fair use" of Our 
+    Trademarks, i.e., merely identifying that you are talking about
+    OpenSSL in a text, without suggesting affiliation, sponsorship or 
+    endorsement.
+3.  You can use Our Trademarks to describe OpenSSL in articles,
+    titles or blog posts.
+4.  You can make t-shirts, desktop wallpapers, caps, or other
```

Comment:
This point should be removed IMO.  We're in the commercial and business usage section and it's hard to see how such could ever be considered non-commercial.

Reply:
This was in the current policy, so probably something that needs to be discussed at OMC.

Reply:
I am ok with this point, however we do not provide a clear definition of what constitutes non-commercial usage. Where I can see a potential for misunderstandings.

Reply:
perfect.

Reply:
Removed